src/node.cc

```diff
@@ -704,6 +704,12 @@ const char *signo_string(int signo) {
 # endif
 #endif
 
+#ifdef SIGINFO
+# if SIGINFO != SIGPWR
```

Comment:
> That defense is not necessary — the C/C++ preprocessors evaluate all undefined symbols to 0.

It emits nasty warnings, however.

> SIGPWR is defined on Linux, but not SIGLOST

Perhaps you mean it's not documented but the define itself exists, it's identical to SIGPWR.